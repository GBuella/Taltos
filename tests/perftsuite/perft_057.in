setboard R6r/8/8/2K5/5k2/8/8/r6R b - - 0 1
echo R6r/8/8/2K5/5k2/8/8/r6R b - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
