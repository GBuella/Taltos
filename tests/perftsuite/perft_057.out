R6r/8/8/2K5/5k2/8/8/r6R b - - 0 1
36
1027
29227
771368
20521342
524966748
